src/app/view/components/dot-message-display/dot-message-display.component.html

```diff
@@ -0,0 +1,9 @@
+<p-toast>
+    <ng-template let-message pTemplate="message">
+        <i  class="pi"
+            [ngClass]="{'pi-info': message.severity === 'info', 'pi-times': message.severity === 'error', 'pi-exclamation-triangle': message.severity === 'warning'}"
+            style="font-size: 2em">
+        </i>
+        <span innerHTML='{{message.detail}}'></span>
```

Comment:
```<span [innerHTML]="message.detail"></span>```

HTML attr is double quoted all the time. Unless you're doing like in the ngclass